pypit/arwave.py

```diff
@@ -60,6 +60,8 @@ def flex_shift(slf, det, obj_skyspec, arx_skyspec):
               (obj_disp[obj_idx]*(2*np.sqrt(2*np.log(2)))*obj_wid[obj_w][obj_keep])
     #obj_res = (obj_sky.wavelength.value[0]+(obj_disp*obj_cent[obj_w][obj_keep]))/(
     #    obj_disp*(2*np.sqrt(2*np.log(2)))*obj_wid[obj_w][obj_keep])
+    if np.any(np.isnan(obj_res)):
```

Comment:
Do we know that errors we want to catch will always be `nan`?  I would use `if np.any(np.invert(np.isfinite(obj_res))):`, despite the fact that it's annoyingly long, to make sure to catch `inf` as well.

Reply:
ah, I was debugging something here.
Turned out to be a bad wavelength issue
much higher up the chain.  Have made the
change and am now having this raise an msgs.error()